docs/tutorial/Permutations.ipynb

```diff
@@ -294,23 +290,26 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Overwriting /tmp/tmptm0w938k\n"
+      "Overwriting /var/folders/mw/21b9jb5d1c9_3_z0dq7hpx1m00j_0b/T/tmp3jpcxxrs\n"
      ]
     }
    ],
    "source": [
     "%%writefile {schema.name}\n",
     "{\n",
-    "  \"version\": 3,\n",
+    "  \"version\": 1,\n",
```

Comment:
The schema had been upgraded to v3, but in this PR, it get retrograded to v1 again.